packages/mock-account-provider/app/routes/quotes.ts

```diff
@@ -35,31 +35,30 @@ export async function action({ request }: ActionArgs) {
   const feeStructure = CONFIG.seed.fees[0]
   const asset = CONFIG.seed.asset
   if (receivedQuote.paymentType == PaymentType.FixedDelivery) {
-    // TODO: handle quote fee calculation for different assets/scales
-    if (
-      receivedQuote.sendAmount.assetCode !== asset.code ||
-      receivedQuote.sendAmount.assetScale !== asset.scale
-    ) {
+    // TODO: handle quote fee calculation for different assets
+    if (receivedQuote.sendAmount.assetCode !== asset.code) {
```

Comment:
Should we check if `receivedQuote` properties exist at all?

Reply:
It's the MAP. I don't think it needs these security checks. If the `receivedQuote` does not have its properties, something is broken in the backend and I sure hope we find out about that without the MAP.

Reply:
>It's the MAP. 

:joy: 

- [ ] Tests added/updated (or it's just the MAP)

Reply:
The MAP doesn't even have tests...